server/controllers/items/lib/bundle_owners_to_items.coffee

```diff
@@ -2,6 +2,7 @@ CONFIG = require 'config'
 __ = CONFIG.universalPath
 _ = __.require('builders', 'utils')
 user_ = __.require 'controllers', 'user/lib/user'
+error_ = __.require 'lib', 'error/error'
```

Comment:
ive seen that elsewhere (dont remember). why is it needed if not called?

Reply:
it is called on line 9

Reply:
right ! another untested line...